tldw_Server_API/app/api/v1/endpoints/media.py

```
@@ -1664,10 +1752,48 @@
         else:
             # Update latest safe_metadata only
             dv_id = latest.get('id')
-            with db.transaction():
-                conn = db.get_connection()
-                conn.execute("UPDATE DocumentVersions SET safe_metadata=? WHERE id=? AND deleted=0", (smj, dv_id))
-                conn.commit()
+            with db.transaction() as conn:
+                # Bump version to satisfy sync trigger and refresh last_modified
+                try:
+                    now_ts = datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S.%f')[:-3] + 'Z'
+                except Exception:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)